tests/functional/test_scoping/issue66/assembly_car1.prog

```diff
@@ -0,0 +1,12 @@
+import "mylib/assembly.tasks"
+import "mylib/position.tasks"
+skills{
+skill PEG_IN_THE_HOLE_1 {
```

Comment:
Maybe to indent a content of the section for better readability. Also for other model files.

Reply:
fixed